Foundation/NSHTTPCookieStorage.swift

```diff
@@ -38,7 +38,7 @@ open class HTTPCookieStorage: NSObject {
 
     private static var sharedStorage: HTTPCookieStorage?
     private static var sharedCookieStorages: [String: HTTPCookieStorage] = [:] //for group storage containers
-
+    private var bundleName: String!
```

Comment:
Agreed. I have modified it. I compute the ‘bundleName’ now and pass it to the ‘filePath’ private function for reuse.